pkgs/development/tools/bazel-watcher/default.nix

```diff
@@ -54,7 +48,7 @@ buildBazelPackage rec {
       sed -e '/^FILE:@go_sdk.*/d' -i $bazelOut/external/\@*.marker
     '';
 
-    sha256 = "1iyjvibvlwg980p7nizr6x5v31dyp4a344f0xn839x393583k59d";
+    sha256 = if stdenv.isDarwin then "1zlg8nay3ij0gnq3ip33cz4f33jhkb1y05n5y8w98hcnkmx3m1vs" else "11pimgaxfx0p003zdxjx9fc9whgxh7qmycihakki4hm2n9ys22fq";
```

Comment:
https://github.com/bazelbuild/bazel-gazelle now includes native binaries after adding it to the workspace, so the hashes will be different depending on the architecture.